pkg/kubemark/controller.go

```diff
@@ -251,7 +253,7 @@ func (kubemarkController *KubemarkController) RemoveNodeFromNodeGroup(nodeGroup
 		err = kubemarkController.externalCluster.client.CoreV1().ReplicationControllers(namespaceKubemark).Delete(context.TODO(), pod.ObjectMeta.Labels["name"],
 			metav1.DeleteOptions{PropagationPolicy: &policy})
 		if err == nil {
-			klog.Infof("marking node %s for deletion", node)
+			klog.InfoS("Marking Node for deletion", "node", node)
```

Comment:
nit: `Node` -> `node`

Reply:
Only this small nit.
Would you update the PR?

Reply:
Fixed.